op-service/txmgr/txmgr_test.go

```diff
@@ -922,3 +923,53 @@ func TestNonceReset(t *testing.T) {
 	// internal nonce tracking should be reset every 3rd tx
 	require.Equal(t, []uint64{0, 0, 1, 2, 0, 1, 2, 0}, nonces)
 }
+
+type mockKms struct {
+	addr       common.Address
+	getAddrErr error
+
+	tx      *types.Transaction
+	signErr error
+}
+
+func (kms *mockKms) GetAddr() (common.Address, error) {
+	return kms.addr, kms.getAddrErr
+}
+
+func (kms *mockKms) SetGetAddr(addr common.Address, err error) {
+	kms.addr = addr
+	kms.getAddrErr = err
+}
+
+func (kms *mockKms) Sign(chainID *big.Int, tx *types.Transaction) (*types.Transaction, error) {
```

Comment:
Nit, but having `Set` methods for an internal mock is kind of weird -- why not simply set the struct fields directly?

Reply:
Removed